### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rnaughtonwk**
Replaced 2 fuzzy images with newer images for documentation.

**JIRA Ticket**: (link)

# What does this Pull Request do?

Updates the images in the readme file.

# What's new?
* Changed two images.

# How should this be tested?
* Confirm that the new images are not blurry.

# Interested parties
@Islandora/7-x-1-x-committers @DonRichards

### CODE REVIEW

**Review by DiegoPino — CHANGES_REQUESTED** [review]
Hey Robin. Seems like the screenshots are pre "redirect to fist child" pull request so missing an option. Could you verify you are actually using the latest version in your Virtual Machine?  or can you there do a git pull? Also, once that is fixed, could you make a pull against 7.x-1.10 too? Ping me if you need anything.

**Comment by rosiel** [comment]
However, this pull request would be great against 7.x-1.10, because that new feature that Diego mentioned isn't in 7.x-1.10. 

Thanks for improving the screenshots!

**Comment by DiegoPino** [comment]
@rnaughtonwk this pull request is sadly complicated: here is the thing:
the 7.x branch, the one you are pulling against in this pull request has an extra functionality, which was merged after the 7.x-1.10 code freeze. That one is the "redirect to first child" one option in the admin form

redirect



7.x-1.10, the release branch has that extra option **_not_**, so your screenshots (your new, sharp and nice ones) apply for 7.x-1.10, but not for 7.x
So basically you need to make two different pulls (sorry) 😢 

let me know if that makes sense and how can I help you

**Comment by rnaughtonwk** [comment]
Figure out what happened and will do the pull request against 7.x-1.10.  Thanks for your help @DiegoPino Diego

**Comment by DiegoPino** [comment]
@rnaughtonwk nice just merged #138! If you want you can update this one to match new functionality or you can simply close it and we can deal with this later, most urgent thing was to update 7.x-1.10 and you did it! 😄 @rosiel thoughts?

**Comment by rnaughtonwk** [comment]
Closing request.
